threescale_api/resources.py

```diff
@@ -128,7 +128,9 @@ def __init__(self, *args, entity_name='application_plan', entity_collection='pla
 
     @property
     def url(self) -> str:
-        return self.parent.url + '/application_plans'
+        if type(self.parent) is Service:
```

Comment:
This should be changed because of the split.

Reply:
there was no split needed in the end